src/sb_utils.R

```diff
@@ -29,4 +46,42 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(files, sb_id, task_yml, out_dir) {
+  # Define task table rows
+  tasks <- basename(files)
```

Comment:
If you use `basename` here, you lose the directory that the file lives in when specifying the file to `item_replace_files()`. So "out_data/cars.csv" becomes "cars.csv", which doesn't exist.